ci/ci/github.py

```diff
@@ -132,6 +132,30 @@ def __init__(self, exception, attributes=None):
 DO_NOT_MERGE = {STACKED_PR, WIP}
 
 
+def clone_or_fetch_script(repo):
+    return """
+{ RETRY_FUNCTION_SCRIPT }
+
+function clone() {{
+    dir=$(mktemp -d)
+    git clone {shq(repo)} $dir
+    for x in $(ls -A $dir); do
+        if [ -e "$dir/$x" ]; then mv -- "$dir/$x" ./; fi
```

Comment:
What is the `--` for in the mv command?

Reply:
It indicates to `mv` that the next arguments are the sources and target. If a directory was named `-x` then, without `--`, `mv` would think you sent it the `-x` argument.